lib/screens/screens_by_alert/behaviour.ex

```diff
@@ -5,15 +5,18 @@ defmodule Screens.ScreensByAlert.Behaviour do
   - `start_link(opts)` is called to start the backend by the supervisor.
   - `put_data(screen_id, list(alert_id))` inserts the list of alert_ids visible on a given screen_id
     to the cache.
-  - `get_screens_by_alert(alert_id)` returns all screen_ids that are currently displaying the given alert_id.
-  - `get_screens_last_updated(screen_id)` returns the timestamp that represents the last time a given screen_id was updated.
+  - `get_screens_by_alert(alert_ids)` returns a map associating each requested alert_id with the screen IDs that are currently displaying it.
+  - `get_screens_last_updated(screen_ids)` returns a map associating each requested screen_id with its last-updated Unix timestamp.
 
   """
   @type screen_id :: String.t()
   @type alert_id :: String.t()
   @type timestamp :: integer()
   @type timestamped_screen_id :: {screen_id, timestamp}
```

Comment:
Can this type be removed now that the return is a map?

Reply:
Oh yeah, good point. The timestamps are an implementation detail at this layer. I'll clean that up.